src/assessments/common/property-bag-column-renderer.tsx

```diff
@@ -50,7 +50,8 @@ export function propertyBagColumnRenderer<TPropertyBag extends ColumnValueBag>(
         propertyMap: DictionaryStringTo<string>,
     ) => {
         if (isEmpty(propertyMap)) {
-            return <React.Fragment>{config.defaultValue}</React.Fragment>;
+            const value: any = config.defaultValue;
```

Comment:
Added separate const because when we do {config.defaultValue} under fragment, the compiler is taking this as a object instead of Element. So to fix it we have added a const